salt/metalk8s/orchestrate/solutions/files/configmap/content.jinja

```diff
@@ -0,0 +1,10 @@
+{%- set bootstrap_pillar = salt.saltutil.runner(
+  'pillar.show_pillar',
+  kwarg={'minion': pillar.bootstrap_id})
+%}
+{%- set solutions = bootstrap_pillar.metalk8s.solutions.available %}
```

Comment:
Can't we just use the `pillar` available in the context? It's computed on the Master, so I don't really see the benefit of getting the result from a specific Minion.

Reply:
No, we can't, because with the pillar from the context, we get the value for the master.
This pillar is generated from the mountpoint discovered on the host and since we mount the Solutions volumes in the salt-master container after this orchestrate, the configmap will not be up to date.
The perfect solution would be:
   - run metalk8s.solutions.available
   - reconfigure salt master
   - update metalk8s-solutions configmap
But it means restarting the salt-master in an orchestrate and AFAIK, it's not doable.

Reply:
Are you sure ? Because from the ext_pillar look like he run a command it self on the bootstrap minion, so should not really need to have a solution mounted in the salt-master container for this

Reply:
@TeddyAndrieux just checked, you're right... so that's weird, without this, my pillar is not up to date.
At the beginning I was using `pillar.metalk8s.solutions.available`, but this was not working...
Maybe it's not related to this, I will check further.

Reply:
bootstrap pillar was refreshed, but not salt-master one... it's fixed !